entity-management-web/src/main/java/eu/europeana/entitymanagement/web/EMController.java

```diff
@@ -106,6 +106,7 @@ public EMController(EntityRecordService entityRecordService,
 	    String etag = computeEtag(timestamp, FormatTypes.jsonld.name(), getApiVersion());
 		checkIfMatchHeaderWithQuotes(etag, request);
 	    entityRecordService.disableEntityRecord(entityRecord);
+			solrService.deleteById(entityRecord.getEntityId());
```

Comment:
This operation will need to be handled similar to a transaction. We will need to do exception handling here. If the entity is disabled in mongo but fails to be deleted in solr we probably want to treat it as as a warning.